webapp/app/forms/steps/unlock_code_activation_steps.py

```diff
@@ -34,7 +28,27 @@ def __init__(self, **kwargs):
     def render(self, data, render_info):
         render_info.form.first_field = next(iter(render_info.form))
         return render_template(
-            'unlock_code/login.html',
+            'react_component.html',
+            component='LoginPage',
+            props=humps.camelize({
+                'step_header': {
+                    'title': render_info.step_title,
+                    'intro': render_info.step_intro,
+                },
+                'form': {
+                    'action': render_info.submit_url,
+                    'csrf_token': generate_csrf(),
+                    'show_overview_button': bool(render_info.overview_url),
+                },
+                'fields': {
+                    field.name: {
+                        'value': field._value(),
+                        'errors': field.errors,
+                    }
+                    for field in render_info.form
+                },
+            }),
+            # TODO: These are still required by base.html to set the page header.
             form=render_info.form,
             render_info=render_info,
             header_title=self.header_title
```

Comment:
Is that true? I don't think we need the render_info in the base.html if we have a header_title - which we explicitly set here.

Reply:
Good point. I took one look at how the page title is set in `base.html` and balked immediately at what I saw. 😆 I'll revisit this.

Reply:
Maybe I should look at how hard it is to explicitly set the `header_title` everywhere so we can get rid of that ugly logic in the template.